backend/FwLite/MiniLcm/Validators/PartOfSpeechIdValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+using MiniLcm.Models;
+
+namespace MiniLcm.Validators;
+
+public class PartOfSpeechIdValidator : AbstractValidator<Guid?>
+{
+    public PartOfSpeechIdValidator()
+    {
+        RuleFor(id => id).Must(BeCanonicalGuid);
+    }
+
+    private bool BeCanonicalGuid(Guid? id)
+    {
+        // TODO: Load GOLDEtic.xml into app as a resource and add singleton providing access to it, then look up GUIDs there
```

Comment:
I'm fine to leave as is for this PR until we figure out how we want to include this data

Reply:
Yeah, GUIDs haven't changed since 2017 so hardcoding a list of GUIDs is very low-risk for future work.